test.js

```diff
@@ -1,12 +1,58 @@
+/* globals describe, it */
+var sleep = require('.');
+var assert = require('assert');
 
-var sleep = require('./');
+function assertApproxEqual(val1, val2) {
+  var epsilon = 1; // we require accuracy to the nearest millisecond
+  var diff = Math.abs(val1 - val2);
+  assert.ok(diff <= epsilon);
+}
 
-console.log('sleeping for 1 seconds...');
-sleep.sleep(1);
-console.log('done');
+describe('sleep', function () {
+  it('works for normal input', function () {
+    var sleepTime = 1;
+    var start = new Date();
+    sleep.sleep(sleepTime);
+    var end = new Date();
+    assertApproxEqual(end - start, sleepTime * 1000);
+  });
 
+  it('works for zero', function () {
+    var sleepTime = 0;
+    var start = new Date();
+    sleep.sleep(sleepTime);
+    var end = new Date();
+    assertApproxEqual(end - start, sleepTime * 1000);
+  });
 
-console.log('sleeping for 2000000 microseconds (2 seconds)');
-sleep.usleep(2000000);
-console.log('done');
+  it('does not allow negative numbers', function () {
```

Comment:
I'm not sure if this is by design, but this is the behavior I observed, and it made sense, so I figured I would put in a unit test for it.